### ISSUE TITLE
Accessibility: Single product tabs missing aria-selected

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43607]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

The tabs on the product single are missing [aria-selected](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected), so screen reader users do not know which tab is currently active.
See: 
https://github.com/woocommerce/woocommerce/blob/758df4854d7b627fc26eb5c0860dc492dea6a06e/plugins/woocommerce/templates/single-product/tabs/tabs.php#L36

The selected tab in a tablist should have aria-selected="true" set. All inactive tabs in the tablist should have aria-selected="false" set. Use JS to change the value of the attribute as tabs are selected.

### Expected behavior

Screen reader users should hear which tab is currently active when viewing tabbed content.

### Actual behavior

Screen reader users do not know which panel is currently active.

### Steps to reproduce

Go to the code linked above and observe that aria-selected is not present.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by dinhtungdu** [comment]
I can confirm the bug @amberhinds. Thank you so much! I added this to our backlog to work on it later.

**Comment by amesplant** [comment]
I am looking into this one
